File: Assignment_3/Assignment3_2.py
"""-------------------------------------------------------------------------------------------
                          Assignment3_2
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
---------------------------------------------------------------------------------------------
Problem statement : This program accepts N elements from the user and 
                    store it into a list and returns the maximum number 
                    from the list
------------------------------------------------------------------------------------------"""
from MarvellousFun import acceptElements
#-------------------------------------------------------------------------------------
# This Function finds the maximum number from the list
#-------------------------------------------------------------------------------------
def findMaximumElement(numberList):
    max = 0
    for num in numberList:
         if num > max :
             max = num
    print("-------------------------------------------------------------------")
    print(f"Maximum element from the list {numberList} is : {max}")     
    print("-------------------------------------------------------------------")
#------------------------------------------------------------------------------------------
# This function accepts a number of elements number from user and calls function 
# acceptElements(number),findMaximumElement(inputNumberList)
#------------------------------------------------------------------------------------------

def main():
    noOfElements=int(input("Enter number of elements in the list:"))
    inputNumberList=acceptElements(noOfElements)
    findMaximumElement(inputNumberList)
#-----------------------------------------------------------------------------------
# This is main entry point of the program
#-----------------------------------------------------------------------------------
if __name__=="__main__":
    main()
#-----------------------------------------------------------------------------------

File: JavaPractice/Assignment_3/Assignment3_2.java
/* -------------------------------------------------------------------------------------------
                          Assignment3_2
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
---------------------------------------------------------------------------------------------
Problem statement : This program accepts N elements from the user and 
                    store it into a list and returns the maximum number 
                    from the list
-----------------------------------------------------------------------------------------  */


import java.util.ArrayList;


public class Assignment3_2 {

	public static void main(String args[]) {

		ArrayList<Integer> numberList=Module.acceptNumbers();
		System.out.println("List Elements are:"+numberList);
		findMaximumNumber(numberList);



	}

	private static void findMaximumNumber(ArrayList<Integer> numberList) {
		int maxNumber=0;
		for(int cnt=0;cnt<numberList.size();cnt++) {
			int element=numberList.get(cnt);

			if(maxNumber<element) {

				maxNumber=element;
			}
		}
		System.out.println("Maximum element is :"+maxNumber);
	}

}
